src/components/flippable/FlippableConnected.tsx

```diff
@@ -0,0 +1,30 @@
+import {connect} from 'react-redux';
+import * as _ from 'underscore';
+
+import {IReactVaporState, IReduxAction, IReduxActionsPayload, ReduxUtils} from '../../Index';
+import {
+    Flippable,
+    IFlippableDispatchProps,
+    IFlippableOwnProps,
+    IFlippableProps,
+    IFlippableStateProps,
+} from './Flippable';
+import {addFlippable, flip, removeFlippable, unflip} from './FlippableActions';
+import {IFlippableState} from './FlippableReducers';
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IFlippableOwnProps): IFlippableStateProps => {
+    const flippable: IFlippableState = _.findWhere(state.flippables, {id: ownProps.id});
+    return {
+        isFlipped: flippable && flippable.flipped,
+    };
+};
+
+const mapDispatchToProps = (dispatch: (action: IReduxAction<IReduxActionsPayload>) => void, ownProps: IFlippableOwnProps): IFlippableDispatchProps => ({

```

Comment:
This line is getting a little long IMO

Reply:
You can use `dispatch: IDispatch` instead of `dispatch: (action: IReduxAction<IReduxActionsPayload>) => void`